lib/archive_auth_mirror/scripts/mirror_archive.py

```diff
@@ -14,17 +15,53 @@
 def main():
     logger = setup_logger(echo=True)
     paths = get_paths()
-    suite = get_config(paths['config'])['suite']
+    config = get_config(paths['config'])
+    suite = config['suite']
+    other_units = config['ssh-peers'].keys()
+    # Using StrictHostKeyChecking=no isn't ideal, but we don't yet
+    # popolate known_hosts with the right keys. But we trust the
+    # network, and we don't push any sensitive data.
+    rsh = 'ssh -o StrictHostKeyChecking=no -i {}'.format(paths['ssh-key'])
 
     logger.info('starting mirroring')
 
     reprepro = Reprepro(logger)
     try:
+        logger.info('fetching new pool packages')
         reprepro.execute(
             '--show-percent', '--export=never', '--keepunreferencedfiles',
             'update', suite)
+
+        logger.info('rsyncing new pool packages to peer units')
+        rsync_multi(paths['static'] / 'ubuntu', other_units, logger, rsh=rsh)
```

Comment:
Purely for readability, you could define a local function like:

```python
def remote_sync(path, delete=False):
    rsync_multi(path, other_units, logger, rsh=rsh, delete=delete)
```
so calls are then shorter.
not a blocking point in any way, though